Zendesk chat v2 documentation (#927)
rudderlabs/rudderstack-docs: docs/sources/extract/zendesk-chat-v2.mdx

<div class="warningBlock">
All the Cloud Extract sources support sending data only to a <Link to="/destinations/warehouse-destinations/">data warehouse destination</Link>.
</div>

<GhBadge
  label={'Stability'}
  message={'Beta'}
  color={'blueviolet'}
  logo={'github'}
/>

## Getting started

To set up Zendesk Chat as a source in RudderStack, follow these steps:

1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
2. Go to **Sources** > **New source** > **Cloud Extract** and select **Zendesk Chat** from the list of sources.
3. Assign a name to your source and click **Continue**.

### Connection settings

To set up Zendesk Chat as a Cloud Extract source, you need to configure the following settings:

<img src="../../assets/cloud-extract-sources/zendesk-chat-connection-settings.png" alt="Zendesk Chat connection settings" />

- **Start Date**: Choose the start date from which you want RudderStack to ingest the Zendesk Chat data. RudderStack will not replicate any data before this date.

<div class="warningBlock">

> If you do not specify the start date, RudderStack will replicate all your data by default.

Reviewer: If they don't provide a starting date, they won't be able to press continue and create the source.

> If you do not specify the start date, RudderStack will replicate all your data by default.

Author: ```suggestion
You will not be able to proceed if you do not specify the start date.
```

</div>

> - **Subdomain**: Enter the subdomain from your Zendesk account's URL. For example, if your Zendesk account URL is `sample.zendesk.com`, the subdomain would be `sample`.

Reviewer: It is **mandatory** to provide a subdomain if their zendesk url follows this format `<subdomain>.zendesk.com`. 
If they have only a zendesk chat account and the url doesn't follow this format they may connect without a subdomain.

Author: Thanks @am6010 for these useful inputs! I've incorporated these pointers in the doc.
